reactor-pool/src/main/java/reactor/pool/InstrumentedPool.java

```diff
@@ -113,7 +113,7 @@ default long secondsSinceLastInteraction() {
 		 */
 		default boolean isInactiveForMoreThan(Duration duration) {
 			return acquiredSize() == 0 && idleSize() == 0 && pendingAcquireSize() == 0 && allocatedSize() == 0
-					&& secondsSinceLastInteraction() >= duration.toMillis();
+					&& !Duration.ofSeconds(secondsSinceLastInteraction()).minus(duration).isNegative();
```

Comment:
Negation of negation is a bit difficult to follow, how about this?

    && duration.minusSeconds(secondsSinceLastInteraction()).isNegative()

Reply:
After tracing the history with @violetagg it seems this implementation is compliant with the javadoc and the other implementation, so it makes sense to keep it consistent. Please disregard my comment.